src/components/code_snippets/_gha-semgrep-app-ssc.mdx

```diff
@@ -22,6 +22,9 @@ on:
     - cron: '20 17 * * *' # Sets Semgrep to scan every day at 17:20 UTC.
     # It is recommended to change the schedule to a random time.
 
+permissions:
```

Comment:
I could be wrong, but I don't think we actually use this snippet, so maybe we should delete it?